pkg/tests/ossm/smcp_annotation_test.go

```diff
@@ -36,99 +40,92 @@ var (
 	//go:embed yaml/deployment-testenv-p.yaml
 	testenvDeploymentP string
 
-	InjectedAnnotationsSMCPPatch = `
-spec:
-  proxy:
-    injection:
-      autoInject: true
-      injectedAnnotations:
-        test1.annotation-from-smcp: test1
-        test2.annotation-from-smcp: '["test2"]'
-        test3.annotation-from-smcp: '{test3}'
-`
+	InjectedAnnotationsSMCPPatch = `{"spec":{"proxy":{"injection":{"autoInject":true,"injectedAnnotations":{"test1.annotation-from-smcp":"test1","test2.annotation-from-smcp":"[\"test2\"]","test3.annotation-from-smcp":"{test3}"}}}}}`
```

Comment:
Is there any benefit to having this in a variable here instead of in the code directly? When you read the code, you now have to go here to see what exactly the patch does.

Reply:
Done